corehq/apps/accounting/invoicing.py

```diff
@@ -491,6 +491,17 @@ def unit_cost(self):
 class UserLineItemFactory(FeatureLineItemFactory):
 
     @property
+    def unit_cost(self):
+        non_prorated_unit_cost = super(UserLineItemFactory, self).unit_cost
+        if self.is_prorated:
+            return Decimal(
+                "%.2f" % round(
+                    non_prorated_unit_cost * self.num_prorated_days / self._days_in_billing_period, 2
```

Comment:
turns out the ```1.0 *``` produced an unsupported operator error (mixing ```float``` and ```Decimal```) so I decided to just assert that ```non_prorated_unit_cost``` is of type ```Decimal```